mangaki/mangaki/models.py

```diff
@@ -229,7 +229,7 @@ def __str__(self):
 
 class WorkTitle(models.Model):
     work = models.ForeignKey('Work')
-    title = models.CharField(max_length=128, blank=True, db_index=True)
+    title = models.CharField(max_length=300, blank=True, db_index=True)  # 300 is safe I hope, we have seen titles of 187 characters.
```

Comment:
Let's use `max_length=255` + please put that comment above the attribute in order to avoid a super long line.